pyproject.toml

```
@@ -0,0 +1,52 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# Testing tools configuration
+[tool.coverage.run]
+branch = true
+
+[tool.coverage.report]
+show_missing = true
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+log_cli_level = "INFO"
+
+# Formatting tools configuration
+[tool.black]
+line-length = 99
+target-version = ["py38"]
+
+[tool.isort]
+profile = "black"
+
+[tool.poetry]
+name = "zookeeper-operator"
+version = "1.0"
+description = "zookeeper-operator"
+authors = [""]
+
+[tool.poetry.dependencies]
+python = ">=3.8"
+black = "^22.3.0"
+kazoo = "^2.8.0"
+ops = "1.5.0"
+pure-sasl = "^0.6.2"
```

Those two packages seems to be related to zookeeper.
```suggestion
ops = "1.5.0"
```